csharp/ql/integration-tests/posix-only/standalone_dependencies_no_framework/test.py

```diff
@@ -1,5 +1,12 @@
-from create_database_utils import *
+import runs_on
+import pytest
 import os
 
-os.environ["CODEQL_EXTRACTOR_CSHARP_BUILDLESS_DOTNET_FRAMEWORK_REFERENCES"] = "/non-existent-path"
-run_codeql_database_create([], lang="csharp", extra_args=["--build-mode=none"])
+
+# Skipping the test on the ARM runners, as we're running into trouble with Mono and nuget.
+@pytest.mark.only_if(runs_on.linux or (runs_on.macos and runs_on.x86_64))
```

Comment:
because of python's `or` and `and` semantics (`a and b` is the same as `b if a else a`, `a or b` the same as `a if a else b`), you can also write
```suggestion
@(runs_on.linux or (runs_on.macos and runs_on.x86_64))
```

Reply:
I have a slight preference to the existing code, if you don't mind :)